corporate/migrations/0001_squashed_0044_convert_ids_to_bigints.py

```diff
@@ -0,0 +1,375 @@
+                    ),
+                ),
+                (
+                    "org_type",
+                    models.PositiveSmallIntegerField(
+                        choices=[
+                            (0, "Unspecified"),
+                            (10, "Business"),
+                            (20, "Open-source project"),
+                            (30, "Education (non-profit)"),
+                            (35, "Education (for-profit)"),
+                            (40, "Research"),
+                            (50, "Event or conference"),
+                            (60, "Non-profit (registered)"),
+                            (70, "Government"),
+                            (80, "Political group"),
+                            (90, "Community"),
+                            (100, "Personal"),
+                            (1000, "Other"),
+                        ],
+                        default=0,
+                    ),
+                ),
+                ("org_website", models.URLField(blank=True, null=True)),
+                ("org_description", models.TextField(default="")),
+                (
+                    "requested_by",
+                    models.ForeignKey(
+                        blank=True,
+                        null=True,
+                        on_delete=django.db.models.deletion.CASCADE,
+                        to=settings.AUTH_USER_MODEL,
+                    ),
+                ),
+                ("expected_total_users", models.TextField(default="")),
+                ("paid_users_count", models.TextField(default="")),
+                ("paid_users_description", models.TextField(default="")),
+                (
+                    "customer",
+                    models.ForeignKey(
+                        on_delete=django.db.models.deletion.CASCADE, to="corporate.customer"
+                    ),
+                ),
+                (
+                    "requested_plan",
+                    models.CharField(
+                        choices=[
+                            ("", "UNSPECIFIED"),
+                            ("Community", "COMMUNITY"),
+                            ("Basic", "BASIC"),
+                            ("Business", "BUSINESS"),
+                        ],
+                        default="",
+                        max_length=50,
+                    ),
+                ),
+            ],
+        ),
+        # Cannot be squashed, because it's a reverse foreign key.
+        migrations.AddField(
+            model_name="customerplan",
+            name="invoiced_through",
+            field=models.ForeignKey(
+                null=True,
+                on_delete=django.db.models.deletion.CASCADE,
+                related_name="+",
+                to="corporate.LicenseLedger",
+            ),
+        ),
+        # Cannot be squashed, because it uses CheckConstraint on other tables.
```

Comment:
I'm not sure I follow?  Those other tables are all from other apps, even.

Reply:
Yeah, I don't know why the Django tool can't squash it. Will improve the comment. We can always hand-squash it later if that doesn't fail for some reason.